curriculum/challenges/english/25-front-end-development/review-semantic-html/671a83934b61f64cefe87a61.md

```diff
@@ -9,7 +9,32 @@ dashedName: review-semantic-html
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Importance of Semantic HTML
 
+- **structural hierarchy for heading elements**: It is important to use the correct heading element to maintain the structural hierarchy of the content. The `h1` element is the highest level of heading and the `h6` element is the lowest level of heading.
+- **presentational HTML elements**: Elements that define the appearance of content. Ex. the deprecated `center`, `big` and `font` elements.
+- **semantic HTML elements**: Elements that hold meaning and structure. Ex. `header`, `nav`, `figure`.
+
+## Semantic HTML Elements
+
+- **`Emphasis` element**: marks text that has stress emphasis.
+- **`Idiomatic Text` element**: used for highlighting alternative voice or mood, idiomatic terms from another language, technical terms, and thoughts.
+- **`Strong Importance` element**: marks text that has strong importance.
```

Comment:
as this are common names for the elements and not how they are coded, using backticks makes it impossible to translate them, I am not sure it's a good idea to have the backtciks here